p2p/configuration_test.go

```diff
@@ -59,3 +59,19 @@ func TestConfiguration_Check(t *testing.T) {
 		})
 	}
 }
+
+func TestDefaultP2PConfiguration(t *testing.T) {
+	tests := []struct {
+		name  string
+		wantC Configuration
+	}{
+		// TODO: Add test cases.
```

Comment:
hm can you add at least one test case? Otherwise I am not sure it's very useful to add this test at all. But is a test of the default value necessary? What kind of errors would it catch?

Reply:
This was an auto-generated test block, I removed it.